riot/val/client.go

```diff
@@ -0,0 +1,21 @@
+package val
+
+import "github.com/KnutZuidema/golio/internal"
+
+// Client pools all methods for endpoints of the Valorant API.
+type Client struct {
+	Content *ContentClient
+	Status  *StatusClient
+	Ranked  *RankedClient
+	Match   *MatchClient
+}
+
+// NewClient returns a new instance of a League of Legends client.
+func NewClient(base *internal.Client) *Client {
```

Comment:
this constructor is not covered by the test suite. add a test for this constructor.